### PR TITLE
[WIP] Identify working tests not requiring setup of a node

### PR DESCRIPTION
**Pull request by lucafavatella**
In scope of https://www.pivotaltracker.com/story/show/164548714

This PR aims at identifying the tests that are known to be working, so that those can be scripted for regression testing and the test coverage can be assessed.

I consider identifying state of tests requiring setup of a node out of scope for this PR: I plan to address those (i.e. `tx_primops_eqc` and `txs_oldnew_eqc`) in following PR.

TODO:
* [x] Assess state of any remaining test not requiring setup of a node:
  * [x] `txs_hardfork_eqc`
* [ ] ~Port history of [oracles tests](https://github.com/aeternity/aeternity/tree/d9521c3863395d3a74a4f38f06cc788095537611/apps/aeoracle/eqc) over to this repo.~
  * ~See https://github.com/Quviq/epoch-eqc/pull/10~

### CODE REVIEW

File: aecore_eqc/txs_eqc.erl [hunk 1]
```diff
@@ -1434,6 +1437,21 @@ prop_txs(Fork) ->
                                            {total, Total == 0 orelse equals(Total, ?PatronAmount - FeeTotal)}]))))))))
     end)))).
 
+load_app_if_unloaded(App) ->
```
Comment (ThomasArts) [inline aecore_eqc/txs_eqc.erl]: far too verbose.... this is test code. Just load it no optimizations

File: aecore_eqc/txs_eqc.erl [hunk 2]
```diff
@@ -1893,7 +1911,9 @@ gen_ttl() ->
 
 %% Generate a contract
 gen_contract() ->
-    elements(["identity", "authorize_nonce"]).
+    elements([ "identity"
+             %% , "authorize_nonce" TODO Restore `authorize_nonce.aes` alongside `identity.aes` in https://github.com/aeternity/aeternity/tree/master/test/contracts
```
Comment (ThomasArts) [inline aecore_eqc/txs_eqc.erl]: should now be in PR

File: aecore_eqc/txs_glue_eqc.erl [hunk 3]
```diff
@@ -224,13 +224,13 @@ spend_args(#{accounts := Accounts, height := Height} = S) ->
                    account -> {ReceiverTag, Receiver#account.key};
                    {name, Name} -> {name, Name}
                end,
-               #{sender_id => aser_id:create(account, Sender#account.key),  %% The sender is asserted to never be a name.
+               #{sender_id => aeser_id:create(account, Sender#account.key),  %% The sender is asserted to never be a name.
```
Comment (ThomasArts) [inline aecore_eqc/txs_glue_eqc.erl:227]: good catch.. this could never have worked....

**Review by ThomasArts — CHANGES_REQUESTED** [review]
Cannot take the complete PR in, but will fix a few things, such as to make more rotten properties work again.

File: aecore_eqc/txs_eqc.erl (hunk 2 above)
Comment (lucafavatella) [reply aecore_eqc/txs_eqc.erl]: I see https://github.com/aeternity/aeternity/pull/2315

File: aecore_eqc/txs_eqc.erl (hunk 1 above)
Comment (lucafavatella) [reply aecore_eqc/txs_eqc.erl]: Ok - I will make load less verbose.

File: aecore_eqc/txs_eqc.erl (hunk 2 above)
Comment (lucafavatella) [reply aecore_eqc/txs_eqc.erl]: So I need to adapt this PR where authorize_nonce used.

File: aecore_eqc/txs_eqc.erl (hunk 1 above)
Comment (lucafavatella) [reply aecore_eqc/txs_eqc.erl]: Addressed in 29164569fc8043b5e34e50797d123d765c314cea

File: README.md [hunk 4]
```diff
@@ -3,6 +3,6 @@
 ### Suggested workflow
  * Check out this repo into e.g. `epoch/eqc`
  * Start a shell using rebar3 (from `epoch/`): `./rebar3 as test shell --apps=""`
- * Move into the directory containing the models you would like to run, e.g.: `1> cd("eqc/aeutils").`
+ * Move into the directory containing the models you would like to run, e.g.: `1> cd("eqc/aeutils_eqc").`
```
Comment (lucafavatella) [inline README.md:6]: I see this doc rename included in https://github.com/Quviq/epoch-eqc/commit/e2b1542f77c1511c5fc400861877b0c26bd27193
Doc update proposed as https://github.com/Quviq/epoch-eqc/pull/9

**Comment by lucafavatella** [comment]
Re 002d7c7
test `txs_eqc` is now fixed by https://github.com/Quviq/epoch-eqc/commit/8f45c985438e986d473c3a87af9464e3571b2b24

**Comment by lucafavatella** [comment]
`txs_glue` test fix split as #11

**Comment by lucafavatella** [comment]
`aec_pow` test fix split as #12

**Comment by lucafavatella** [comment]
Mostly cherry-picked and merged, or being handled as other PRs e.g. https://github.com/Quviq/epoch-eqc/pull/26

So closing this.

**Pull request closed without merging**